src/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const footerInfo = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
+const Footer = () => {
+  return (
+    <ul className="footer">
+      {footerInfo.map(info => (
+        <li>{info}</li>
+      ))}
+    </ul>
```

Comment:
Remember to use a key whenever you map in React, noticed you have used them elsewhere, which is good!